### ISSUE TITLE
PpictorialBar Series Label- rotate, When I rotate the Data label of bar series then same direction i wan to rotate the Pictorial Bar label icon

### ISSUE THREAD

**Issue by sanganasrikanth9618** [apache/echarts#19650]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=bar-simple&code=G4QwTgBArgDgCiALgCwgXgFAQgchk5ALgHpiBZADgogCYA2AFgBsBmARgnYjZpYFoeNCAAZadagwYiRfegE5uvRVJY0-qgQHZNWugC0cAbgyhIAEwDuCFOiy58KEuUkd6zdWvYClPFjPoUfJLSogHe_L7qHFxsmnTccQbGGAD2MIgAlikAdugQAN52AB4AgkUZAM6EBXbYiACeMACm1TgAxkhNAOYpYPU4ADS1EGZIINUA2jhkOYO4ACpQTXM4AOpNZivzyFArAGJgGSsAykgnUNk4ALp2AL5D2PVlldWF2HWNLbigTEs4dw8IBUmocmlUIBNhm93thRohxhCeMIBrRhMjuABWdEUdGadFsNj4ljCK6AmENZqtABG4EGw2wTBAVKaTFe9PeYBS8MQXxoeLJMKByBSFmqiDASwFMJgKQqGUyOVaiDS_0FEFuw3uUPZFXhbQA1q0zBkAGYmsBNbKIeYAORAAFtllKRmNJkiUTQ0Si2FiUTiUfzuITvcTSeyKV88Bk2srDiAmAAhWnOxnM1k1NXYTncr58OTCdlatUVer2qkpdOWazIZ0lssV44ZABeX2DOtL5aYAHkzcDEJMaBiUQIMWHix2K3BZfKstlWpbNpq7DdboYgA

### Steps to Reproduce

1.  I took a simple Bar Chart series, applied label rotation, and then added a pictorial bar series. 
2. Although I also rotated the labels for the pictorial bar, it did not rotate

Options: 

var upPath =
  'path://M88 264l31 31 123-122 0 268 44 0 0-269 123 124 32-32-177-176Z';
var dwPath =
  'path://M441 264l-32-31-123 123 0-268-44 0 0 268-123-123-31 31 176 176Z';

option = {
  xAxis: {
    type: 'category',
    data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
  },
  yAxis: {
    type: 'value'
  },
  series: [
    {
      data: [120, 200, 150, 80, 70, 110, 130],
      type: 'bar',
      label: {
        rotate: 270,
        show: true,
        position: 'top'
      }
    },
    {
      stack: 'diffrentTName',
      data: [120, 200, 150, 80, 70, 110, 130],
      type: 'pictorialBar',
      label: {
        rotate: -90
      },
      symbol: dwPath,
      symbolSize: 10,
      symbolOffset: [25, -15],
      symbolPosition: 'end'
    }
  ]
};
![pIctorialBar](https://github.com/apache/echarts/assets/144200834/3c200f91-8916-4930-9b0a-4f844bc43034)
and 

### Current Behavior

Pictorial bar series labels are not rotating

### Expected Behavior

Pictorial bar series labels should rotate.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by MatthiasMert** [comment]
Maybe you should use [symbolRotate](https://echarts.apache.org/en/option.html#series-pictorialBar.symbolRotate) to rotate the symbo, instead of the [label](https://echarts.apache.org/en/option.html#series-pictorialBar.label.show) whis visability is set to false by default.

**Comment by sanganasrikanth9618** [comment]
Yes, the above symbolRotate worked, thank you. However, when I rotated both the bar label and the Pictorial bar to 270 degrees, the Pictorial bar label displayed strangely. I have uploaded the issue behavior and the expected behavior below.

Options: 
![options](https://github.com/apache/echarts/assets/144200834/d00a8d28-2019-48cf-b831-433e5493a63c)

With the above options, the Pictorial bar icon rotated correctly, but the pictorial bar icon does not respect the label color. 
Issue Image: 
![Current ](https://github.com/apache/echarts/assets/144200834/e4bd2f8b-badc-40a9-ae4b-72a58dcca4e0)

Expected Behavior: 
![Expected Behavior](https://github.com/apache/echarts/assets/144200834/bc4f2194-61f1-4a93-bf85-2d4f7bbbca4e)

Instead of adjusting the **symbolOffset,** is there any other way to ensure that when I change the rotate and symbol rotate properties, both the bar label and Pictorial bar label display correctly?

**Comment by MatthiasMert** [comment]
You are confusing the label of the `bar` series with the label of the `pictorialBar` series. These are two completely different series instances that do not share any data. You just put the same data into both.

That beeing said, you can align the label of the `pictorialBar` series with its symbol by using [align](https://echarts.apache.org/en/option.html#series-pictorialBar.label.align), [verticalAlign](https://echarts.apache.org/en/option.html#series-pictorialBar.label.verticalAlign), [distance](https://echarts.apache.org/en/option.html#series-pictorialBar.label.distance), [position](https://echarts.apache.org/en/option.html#series-pictorialBar.label.position) and [offset](https://echarts.apache.org/en/option.html#series-pictorialBar.label.offset). To get the effect you are looking for this seems to do the job:

```
position: 'top',
verticalAlign: 'middle',
distance: -25
```

[Example](https://echarts.apache.org/examples/en/editor.html?c=bar-simple&code=G4QwTgBArgDgCiALgCwgXgFAQgchk5ALgHpiBZADgogCYA2AFgBsBmARgnYjZpYFoeNCAAZadagwYiRfegE5uvRVJY0-qgQHZNWugC0cAbgyhIAEwDuCFOiy58KEuUkd6zdWvYClPFjPoUfJLSogHe_L7qHFxsmnTccQbGGAD2MIgAlikAdugQAN52AB4AgkUZAM6EBXbYiACeMACm1TgAxkhNAOYpYPU4ADS1EGZIINUA2jhkOYO4ACpQTXM4AOpNZivzyFArAGJgGSsAykgnUNk4ALp2AL5D2PVlldWF2HWNLbigTEs4dw8IBUmocmlUIBNhm93thRohxhCeMIBrRhMjuABWdEUdGadFsNj4ljCK6AmENZqtABG4EGw3uUOG2Aq8LaAGtWmYMgAzblgJrZRDzAByIAAtssye84QiJkiUTQ0Si2FiUTiUXjlYTlcTSUyIBSvngMm1EL0MiAmAAhWlS5n1MVUlJMaqWazIO1Ah1OpgAJRS8MQXxomv1FW9zuOGQAXl9tWGI0w4CkKhlMjlWgLNp7w47nQB5XnAxCTdE4ARogCk109TBAVKaLpqMJhYADnWqIeR-uZyBSFmqiDAS092BgKbTWWyrTNMDpLfewBBmQ6TBKTAyXWnuDFGTMZiYkp7I0q8OybS-sgx-oZMNudhut0MQA)


Note, that you can move the symbol horizontally now via [symbolOffset](https://echarts.apache.org/en/option.html#series-pictorialBar.symbolOffset) and the label will keep the distance you specified.
